src/components/ReportingConfig/EmailDeliveryMethodForm.jsx

```diff
@@ -10,12 +10,7 @@ const EmailDeliveryMethodForm = ({ invalidFields, config, handleBlur }) => {
   return (
     <div className="row">
       <div className="col">
-        <ValidationFormGroup
-          for="email"
-          helpText="The email(s), one per line, where the report should be sent"
-          invalidMessage="Required. One email per line. Emails must be formatted properly (email@domain.com)"
-          invalid={invalidFields.emailRaw}
-        >
+        <Form.Group isInvalid={invalidFields.emailRaw}>
           <label htmlFor="email">Email(s)</label>
           <Input
```

Comment:
As depicted in [ENT-8329](https://2u-internal.atlassian.net/browse/ENT-8329), `Input` is deprecated. In this case and many others below, should be migrated to `Form.Control`.

Also note that a nice benefit of `Form.Group`, `Form.Control`, and `Form.Label` here is that it will automatically apply the appropriate `id`, `for`, etc. attributes, etc. without it needing to be explicitly passed anymore.

For example, the `controlId` prop below is applied to the appropriate HTML attributes:

```jsx
  <Form.Group
    isInvalid
    controlId="explicit-id-1"
  >
    <Form.Label>What kind of cats?</Form.Label>
    <Form.Control />
    <Form.Control.Feedback type="invalid">You are incorrect!</Form.Control.Feedback>
  </Form.Group>
```

<img width="832" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/e5c32092-c366-46a6-914f-9e52ef0ba372">

You might also run into some scenarios where as `Input` has a checkbox type; in those cases, `Form.Checkbox` should be used.

**Paragon Docs**

* [`Input`](https://paragon-openedx.netlify.app/components/input/)
* [`Form.Control`](https://paragon-openedx.netlify.app/components/form/form-control/)
* [`Form.Label`](https://paragon-openedx.netlify.app/components/form/form-label/)
* [`Form.Checkbox`](https://paragon-openedx.netlify.app/components/form/form-checkbox/)